src/wp-includes/update.php

```diff
@@ -977,3 +1033,5 @@ function wp_clean_update_cache() {
 add_action( 'wp_maybe_auto_update', 'wp_maybe_auto_update' );
 
 add_action( 'init', 'wp_schedule_update_checks' );
+
+add_action( 'wp_delete_temp_updater_backups', 'wp_delete_all_temp_backups' );
```

Comment:
This and probably others should be in default filters.

Reply:
@peterwilsoncc has this issue been resolved with the instructions on how to fix the VB issue?

https://github.com/costdev/wp-virtualbox-testing

Reply:
I'd like to note that the linked repo and instructions are a non-starter and are unlikely to ever reach more than 1% of all WordPress VirtualBox installs. You should proceed on the assumption it will never be applied, that the user will never be aware it exists, and that it will be beyond their technical abilities to implement if they do discover it